nylas/client/client.py

```diff
@@ -198,6 +199,15 @@ def revoke_all_tokens(self, keep_access_token=None):
             self.auth_token = None
             self.access_token = None
 
+    def ip_addresses(self):
+        ip_addresses_url = self.ip_addresses_url.format(
+            client_id=self.app_id)
+        headers = {"Content-Type": "application/json"}
+        headers.update(self.admin_session.headers)
```

Comment:
Is there a reason why _this_ endpoint needs the `Content-Type` header, but others do not? We should make it consistent.

Reply:
nope - was just following the format I saw for other endpoints. Should I remove it?

Reply:
To be honest, _all_ of the admin endpoints should probably set this header -- but let's not do that in this pull request. Consistency makes code easier to reason about, which is why we want to be consistent here.

I recommend removing the `Content-Type` header from this method, so that it's consistent with all the other methods that use admin endpoints. If you want to go the extra mile, I suggest making a _new_ pull request that sets the `Content-Type` header for _all_ the admin endpoints, and we can discuss it in _that_ pull request. That way, it's consistent and more correct. Reasonable?

Reply:
Sounds good. I'll remove it from this endpoint.